src/main/java/org/springframework/data/elasticsearch/core/index/MappingParameters.java

```diff
@@ -88,6 +90,7 @@
 	private Similarity similarity = Similarity.Default;
 	private TermVector termVector = TermVector.none;
 	private double scalingFactor = 1.0;
+	private int maxShingleSize;
```

Comment:
```suggestion
	private Integer maxShingleSize = null;
```

here we can have a nullable field